lte/gateway/deploy/roles/containers/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+#
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+- name: Include vars of all.yaml
+  include_vars:
+    file: all.yaml
+
+- name: create the env file
+  template:
+    src: env.j2
+    dest: /var/opt/magma/docker/.env
+    mode: 0400
+
+- name: Clone the git repo
+  git:
+   repo: "https://github.com/magma/magma.git"
+   clone: yes
+   force: yes
+   dest: "/opt/magma"
+   version: "{{ image_version }}"
+
+- name: copy the config files for magma
+  shell: cp -r  /opt/magma/lte/gateway/configs/* /etc/magma/
```

Comment:
Would the config files be modified for the containerization use-case? Ex: magmad is not required to monitor any services running on the node anymore.

Reply:
magmad is still able to monitor all the other services.
The only change for now is nat interface in pipelined which is set to a default eth2 (moved to eth0)